frontend/src/components/Sidebar/prepareRoutes.ts

```diff
@@ -240,6 +230,16 @@ function prepareRoutes(
         },
       ],
     },
+    {
+      name: 'crds',
+      label: t('glossary|Custom Resources'),
+      subList: [
+        {
+          name: 'crs',
+          label: t('glossary|CR Instances'),
```

Comment:
Just call these `translation|Instances`. The context of being CR instances is already given by the parent item.